src/screens/OrgList/OrgList.tsx

```
@@ -227,7 +244,7 @@ function orgList(): JSX.Element {
           image: image,
         },
       });
-
+      toggleModal;
```

ESLint

[error] Expected an assignment or function call and instead saw an expression.

(@typescript-eslint/no-unused-expressions)